tfx/components/example_validator/component.py

```diff
@@ -17,16 +17,18 @@
 from __future__ import division
 from __future__ import print_function
 
-from typing import Optional, Text
+from typing import Optional, List, Text
```

Comment:
did you try if dsl compiler pass or not?

Reply:
it works now, dsl compiler test is passed.